src/01-general/users.apib

```diff
@@ -120,3 +120,17 @@ Get details for a single user.
                     + tr-TR
             + function: `Sales` (string)
             + time_zone: `Europe/Brussels` (string)
+
+
+### users.updateRates [POST /users.updateRates]
+
+Update user rates used for timetracking
```

Comment:
I think we should indicate that there's an alternative projectItems.updateInternalRate endpoint that can only update some profit entries. Similarly, in the related endpoint, we should explain that we also have this endpoint that can set a default rate as well.

Reply:
> I think we should indicate that there's an alternative projectItems.updateInternalRate endpoint that can only update some profit entries. Similarly, in the related endpoint, we should explain that we also have this endpoint that can set a default rate as well.

However the documentation for projectItem endpoints is not available yet?

Reply:
True, but I think we should hold this users.updateRates back as well (the concept of internal rate is only visible for users that have the profit feature flag for now). And by linking I don't mean creating a hyperlink, but just mentioning it should be enough for now.